packages/flame/test/components/component_lifecycle_test.dart

```diff
@@ -47,6 +47,47 @@ void main() {
       );
     });
 
+    flameGame.test('component hasMounted completes', (game) async {
```

Comment:
```suggestion
    testWithFlameGame('component hasMounted completes', (game) async {
```

Reply:
All of this file is using an instance, better keep with the flow